src/pages/FavoriteDrinksPage/FavoriteDrinksPage.styled.js

```diff
@@ -1,8 +1,15 @@
 import styled from "styled-components";
 
 export const FavoriteTitle = styled.h2`
-  color: var(--white);
   font-size: 32px;
   font-weight: 600;
   line-height: 1.18;
+
+  @media screen and (min-width: 768px) {
```

Comment:
Можна замінити таким чином:
```suggestion
 @media ${device.tablet} {
```